### ISSUE TITLE
want more recent pokerstove release

### ISSUE THREAD

**Issue by CircleOnCircles** [andrewprock/pokerstove#64]
I am developing a [homebrew formula of pokerstove](https://github.com/CircleOnCircles/homebrew-core/commit/62df55c9c2160d4dc7f2674e97b35944b4fb4c19).

So, macos users will able to 
```bash
brew install pokerstove
```

It will be very nice if you could create the more recent release(only release here is initial release from 2016 - 3 years ago!!).

**Comment by CircleOnCircles** [comment]
anyone who want to try could use

```
brew install circleoncircles/homebrew-tap/pokerstove
```

**Comment by andrewprock** [comment]
nice!

**Comment by CircleOnCircles** [comment]
I don’t see any new release.

**Comment by andrewprock** [comment]
Released as v1.0
